src/components/Textarea/src/TextareaControl.vue

```diff
@@ -98,29 +98,47 @@ export default {
 	until we get a Theme Context component
 */
 .variant_fill {
-	--font-family: inherit;
 	--color-background: rgba(0, 0, 0, 0.05);
 	--color-background-focus: rgba(255, 255, 255, 0.95);
 	--color-foreground: rgba(0, 0, 0, 0.9);
 	--color-placeholder: rgba(0, 0, 0, 0.55);
 	--color-disabled: rgba(0, 0, 0, 0.3);
-	--color-border: #222;
+	--color-background-disabled: rgba(0, 0, 0, 0.05);
+	--color-border: transparent;
+	--color-border-active: #222;
 	--border-radius: 8px;
 	--color-error: #ff3b30;
 	--focus-shadow: 0 0 0 2px rgba(34, 34, 34, 0.3);
 }
 
+.variant_outline {
+	--color-background: #fff;
+	--color-background-focus: #fff;
+	--color-foreground: rgba(0, 0, 0, 0.9);
+	--color-placeholder: rgba(0, 0, 0, 0.55);
```

Comment:
@pretzelhammer are all variables component specific? No shared higher level variables to keep styles consistent between groups of components, e.g. form inputs: text area & input. Or is this a stop gap for theming?

Reply:
Stopgap for theming.